### PR TITLE
Install linters

### PR DESCRIPTION
**Pull request by danielamoreno699**
_In this project, I:_ 

-Installed linters
-Installed React package
-Added Readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,130 @@
...
+- **React**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo Coming Soon <a name="live-demo"></a>
+
+- [Live Demo Link Coming Soon](#)
```
Comment (addod19) [inline README.md]: - [ ] It would be better to remove the misleading live demo link, this makes our README file no professional

File: README.md [hunk 2]
```diff
@@ -1 +1,130 @@
...
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, let us know on any of my social media account ;)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```
Comment (addod19) [inline README.md:111]: - [ ] Kindly add some text under the acknowledgement section, leaving it blank does not help in making our README professional

File: README.md [hunk 3]
```diff
@@ -1 +1,130 @@
...
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https:/github.com/danielamoreno699/my-app-calculator/LICENSE) licensed.
```
Comment (addod19) [inline README.md]: - [ ] Kindly fix the broken MIT license file, it returns `404` error page not found

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @danielamoreno699, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- [ ] No linter errors recorded :100: 
- [ ] App created with `create-react-app` :1st_place_medal: 
- [ ] App runs with any console errors :rocket: 
- [ ] React script looks awesome :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
Hi @danielamoreno699, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Correct Gitflow ✔️ 
- [ ] The app is set up correctly ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**